### ISSUE TITLE
Next Release

### ISSUE THREAD

**Issue by tcs-apeters** [wkhtmltopdf/wkhtmltopdf#3850]
What is the schedule for the next release?

**Comment by scfcode** [comment]
Especially one with the Mac regression ( https://github.com/wkhtmltopdf/wkhtmltopdf/issues/3241 ) fixed please.

**Comment by ashkulz** [comment]
Currently working on a reworked packaging system in the `wkhtmltopdf/packaging` repository, you can download preview builds [for linux](https://builds.wkhtmltopdf.org/0.12.5-dev/). For OS X and Windows, trying to automate the build using vagrant so a release can be created easily.

Planning to release a `-rc` by next weekend, and a release after a week .. so expect May 15 for the 0.12.5 release.

**Comment by saurabh-prohaktiv** [comment]
Any updates on the release?

**Comment by ashkulz** [comment]
Still working on the release process, rc should be out this weekend. Builds for all platforms are available and there won't be changes till final release, unless there is feedback on the rc.

**Comment by ashkulz** [comment]
A release candidate [is available](https://wkhtmltopdf.org/downloads.html#release-candidate) -- please test and report your findings before the final release, which should happen by next weekend.

**Comment by saurabh-prohaktiv** [comment]
I have tested 0.12.5-rc build and as for me it looks good, the dpi option works fine with mac and linux as expected. when is this going to be released?

**Comment by ashkulz** [comment]
Monday, if no major issues are reported with -rc build.

**Comment by saurabh-prohaktiv** [comment]
Ok, Thanks for the updates

**Comment by saurabh-prohaktiv** [comment]
I can see that `0.12.5` has been released, however this release is missing `install.md` file and `scripts` folder which were present in `0.12.4` version, also it is not available at wkhtmltopdf.org/downloads.html, so is there any documentation to compile the latest release from source code?

**Comment by ashkulz** [comment]
The build scripts have moved to the [packaging repository](https://github.com/wkhtmltopdf/wkhtmltopdf) so that the build scripts are orthogonal to the main source repository (and new platforms/distros can be added later on) -- documentation is pending, and will be done as time permits.

This approach has proved its value immediately: I made an mistake in using `~` as part of the version for denoting the platform (see [`-rc` builds](https://builds.wkhtmltopdf.org/0.12.5-rc/)). When uploading to the `0.12.5` release, I discovered that Github doesn't support `~` in the filename and replaces it with `.` -- hence wkhtmltopdf/packaging@e4c51df1646d6855cdc0fb42f458fa2325ed75c7 fixes that and am re-running the build (will take ~14 hours to complete across all platforms).
